scripts/manage-db.py

```diff
@@ -45,6 +45,20 @@ def cleanup_releases(trans, nightly_age, dryrun=True):
         trans.execute("DELETE releases FROM releases" + query)
 
 
+def cleanup_releases_json(trans, nightly_age, dryrun=True):
+    query = RELEASES_CLEANUP_CONDITION.format(nightly_age=nightly_age)
+    if dryrun:
+        todelete = trans.execute("SELECT name FROM releases_json" + query).fetchall()
+        print("Releases JSON rows to be deleted:")
+        if todelete:
+            print("\n".join(itertools.chain(*todelete)))
+        else:
+            print("  - None")
+    else:
+        trans.execute("DELETE releases_json FROM releases_json" + query)
+        trans.execute("DELETE release_assets FROM release_assets" + query)
```

Comment:
Does`DELETE FROM table_name WHERE condition` work? Instead of `DELETE table_name FROM table_name WHERE condition`

Reply:
Looks like it's either `DELETE tbl_name FROM tbl_name LEFT JOIN ...` or `DELETE FROM tbl_name USING tbl_name LEFT JOIN ...`, when multiple tables are involved.
(https://dev.mysql.com/doc/refman/5.7/en/delete.html#idm46005468799856)

Reply:
Oh ok